### ISSUE TITLE
Spell errors are missed

### ISSUE THREAD

**Issue by sverzakov** [streetsidesoftware/vscode-spell-checker#2116]
In the current version version v2.3.4 (and earlier) words "excpetion" and "charecter" are not detected as misspellings.

![image](https://user-images.githubusercontent.com/11227886/182431024-a09c155b-aaca-4fd0-8106-ad73a979e12c.png)

**Comment by Jason3S** [comment]
@sverzakov,

Somewhere in your configuration, `allowCompoundWords` has been set to `true`. I suggest setting it to `false`.

To see if a word exists in a dictionary, use the following command in the terminal (assumes you have node installed):

```sh
npx cspell trace charecter
```

To see if a compound words exists:

```sh
npx cspell trace --allow-compound-words charecter
```

![image](https://user-images.githubusercontent.com/3740137/183238130-1106feed-2db6-4757-8aba-f87a3072fe76.png)
